framework/src/Volo.Abp.Security/Volo/Abp/Security/Claims/AbpClaimTypes.cs

```diff
@@ -13,6 +13,21 @@ public static class AbpClaimTypes
         /// </summary>
         public static string UserName { get; set; } = ClaimTypes.Name;
 
+        /// <summary>
+        /// Default: <see cref="ClaimTypes.GivenName"/>
+        /// </summary>
+        public static string Name { get; set; } = ClaimTypes.GivenName;
+
+        /// <summary>
+        /// Default: <see cref="ClaimTypes.Surname"/>
+        /// </summary>
+        public static string SurName { get; set; } = ClaimTypes.Surname;
+
+        /// <summary>
+        /// Default: security_stamp
+        /// </summary>
+        public static string SecurityStamp { get; set; } =  "security_stamp";
```

Comment:
Do not define such a property. We don't need it.
It is already different than the ASP.NET Core's default (which is `AspNet.Identity.SecurityStamp`). We don't like to change default wherever possible.
So, remove this property and its usage.